src/sfctl/helps/deployment.py

```diff
@@ -0,0 +1,26 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Help documentation for managing Service Fabric Mesh Resources."""
+
+from knack.help_files import helps
+
+helps['mesh deployment create'] = """
+    type: command
+    short-summary: Creates a deployment of Service Fabric Mesh Resources
+    parameters:
+        - name: --input-yaml-file-paths
+          type: string
+          short-summary: Comma separated file paths of all the yaml files or the directory (recursive) which contain yaml files

```

Comment:
Can this be an absolute path as well? Same for --parameters

Reply:
added the same details to help text